### DISCUSSION TITLE
Running minidexed on a touchscreen

### DISCUSSION THREAD

**Discussion by Marklo101** [probonopd/MiniDexed#697] [Q&A] [answered]
Hi all, pretty new to this. A lifelong musician and fan of the various DX’s. I’d love to have one but could never afford to buy a real DX.  But of course Minidexed looks like the answer to that particular dilemma.

I have a PI m3 a waveshape 3.2 touch screen and a sub sound module and I’m wondering if it’s possible to get minidexed up and running in this setup and if so how.

**Comment by diyelectromusic** [comment, marked as answer]
It isn't possible to run MiniDexed and make use of the touchscreen as the UI itself.  If it is a HDMI touchscreen then it can be used as the debug console, but that is all I'm afraid.  I don't know what a sub sound module is, but if you have a link I'll take a look.

Note: this needs a genuine Raspberry Pi, so if by Pi M3 you mean a Banana Pi m3, then the answer is definitely no I'm afraid.

Assuming a genuine Pi, then 0ne other option might be to consider Zynthian. I''ve used a Waveshare 4" HDMI touchscreen with Zynthian, but this is a full synth-oriented OS installation.  It includes a number of Synth engines, including the fuller Dexed on which MiniDexed is based.  Zynthian is quite a bit more resource intensive than MiniDexed though, so it might be ok for just Dexed on a Pi 3, but ideally a Pi 4 would be better.

I have some details for Zynthian here: https://diyelectromusic.com/2022/06/30/zynthian-revisited/

Another, non-Pi option might be MicroDexed touch, but this runs on a Teensy 4.1 with a TFT touch screen - but that is a completely different project of course to your Pi 3 and a Waveshare (I assume its Waveshare not Waveshape) screen :)

A different synth project that can use a HDMI screen is Rene's original minisynth project for circle.  This is using the same bare metal environment as MiniDexed but for a pseudo analog-synth type device.  See: https://github.com/rsta2/minisynth.  My own notes about getting it running can be found here: https://diyelectromusic.com/2022/03/20/bare-metal-raspberry-pi-synth-part-4/

But yes, these are all for a genuine Pi (or Teensy in the case of MicroDexed).

(If you are using a Banana Pi, I don't know if Dexed itself is available for the Banana Pi linux version you're using, but it might be worth a look...)

Kevin

**Reply by Marklo101** [reply]
Hi Kevin,

Thanks for the very detailed reply. Yes it’s a genuine raspberry pi 3. I kind of like the idea of a small standalone synth. Zynthians interesting by bit I have an MPC one and a micro freak for that kind of thing and I don’t want another groovebox. I may just go and buy the display and hang on to the touchscreen for future use.

Again thanks for the reply.
